include/z64camera.h

```diff
@@ -835,56 +841,72 @@ typedef struct {
     { swingPitchAdj, CAM_DATA_SWING_PITCH_ADJ }, \
     { fov, CAM_DATA_FOV }, \
     { atLerpStepScale, CAM_DATA_AT_LERP_STEP_SCALE }, \
-    { yawUpdateRateTarget, CAM_DATA_YAW_UPDATE_RATE_TARGET }, \
+    { initTimer, CAM_DATA_YAW_UPDATE_RATE_TARGET }, \
     { interfaceField, CAM_DATA_INTERFACE_FIELD }
 
 typedef struct {
-    /* 0x00 */ f32 unk_00;
-    /* 0x04 */ f32 unk_04;
-    /* 0x08 */ f32 unk_08;
-    /* 0x0C */ f32 unk_0C;
-    /* 0x10 */ f32 unk_10;
-    /* 0x14 */ f32 unk_14;
-    /* 0x18 */ f32 unk_18;
+    /* 0x00 */ f32 yOffset;
+    /* 0x04 */ f32 eyeDist;
+    /* 0x08 */ f32 pitchTarget; // degrees, usage varies with KEEPON4_FLAG_EYE_
+    /* 0x0C */ f32 yawTarget; // degrees, usage varies with KEEPON4_FLAG_EYE_
+    /* 0x10 */ f32 atOffsetPlayerForwards; // distance to offset at by in the player's forwards direction
```

Comment:
"distance to offset `at` by, in the player's forwards direction"
will help alot with the word soup here. i think putting at and eye in backticks in general helps the reader quickly know its referring to the camera component, and not the regular word